pages/accountLists/[accountListId]/settings/preferences.page.test.tsx

```diff
@@ -108,7 +113,17 @@ const MocksProviders = (props: {
               exportedAt: null,
             },
           },
+          GetUserOptions: {
+            userOptions: [
+              {
+                id: 1,
```

Comment:
`id` is supposed to be a string. It could probably actually be omitted and autogenerated.

```suggestion
                id: 'option-1',
```